### PR TITLE
Heroku documentation .env vars

### PR DESCRIPTION
**Pull request by iamluca**
Forgot in the heroku instance to put the prefix.

### CODE REVIEW

File: app.json [hunk 1]
```diff
@@ -8,13 +8,21 @@
...
+        "description": "Operations center for the server you are serving for.",
+        "required": false
+    },
     "OWNERS": {
         "description": "Comma seperated user IDs of people that are allowed to use owner only commands. (eval and update)",
         "required": true
     },
+    "PREFIX": {
+        "description": "Beginning of command prefix.",
+        "required": true
```
Comment (kyb3r) [inline app.json]: Should not be required, also document that the default prefix is `?`

File: app.json [hunk 2]
```diff
@@ -8,13 +8,21 @@
         "required": true
     },
     "GUILD_ID": {
-        "description": "The id for the server you are hosting this bot for.",
+        "description": "The id for the server you are serving this bot for.",
         "required": true
     },
+    "MODMAIL_GUILD_ID": {
+        "description": "Operations center for the server you are serving for.",
```
Comment (kyb3r) [inline app.json]: Describe that this config var is only necessary if you want the bot to have a separate operations center.

File: app.json (hunk 1 above)
Comment (iamluca) [reply app.json]: Ok. Fixing

File: app.json (hunk 2 above)
Comment (iamluca) [reply app.json]: Fixing

**Comment by iamluca** [comment]
@kyb3r Check now!

File: app.json (hunk 2 above)
Comment (kyb3r) [reply app.json]: Try: `The server which is the operations center for the bot. This is only necessary if you want a separate server in which modmail operations occur.`

File: app.json (hunk 1 above)
Comment (kyb3r) [reply app.json]: You forgot to say that the default prefix is `?`

**Comment by kyb3r** [comment]
Config that is not required at start time should not be in here. You can easily set up a prefix, status, etc with the commands within the bot.

**Pull request closed without merging**